forest/shared/src/cli/client.rs

```diff
@@ -32,6 +32,7 @@ pub struct Client {
     pub metrics_address: SocketAddr,
     /// RPC bind, e.g. 127.0.0.1:1234
     pub rpc_address: SocketAddr,
+    pub download_snapshot: bool,
```

Comment:
It would be nice to expose it. I don't see it available from CLI, except for the configuration file. Or am I missing something?
```
❯ target/release/forest --help
forest-daemon 0.4.1+git.9d6180bb
ChainSafe Systems <info@chainsafe.io>
CLI options

USAGE:
    forest [FLAGS] [OPTIONS] [cmd]

FLAGS:
        --detach               Daemonize Forest process
        --halt-after-import    Halt with exit code 0 after successfully importing a snapshot
    -h, --help                 Prints help information
        --skip-load            Skips loading CAR file and uses header to index chain. Assumes a pre-loaded database
    -V, --version              Prints version information

OPTIONS:
        --chain <chain>
            Choose network chain to sync to [default: mainnet]  [possible values: mainnet, calibnet]

    -c, --config <config>                            A TOML file containing relevant configurations
        --encrypt-keystore <encrypt-keystore>        Encrypt the key-store (default: true)
    -g, --genesis <genesis>                          The genesis CAR file
        --height <height>                            Validate snapshot at given EPOCH
        --import-chain <import-chain>                Import a chain from a local CAR file or URL
        --import-snapshot <import-snapshot>          Import a snapshot from a local CAR file or URL
    -k, --kademlia <kademlia>                        Allow Kademlia (default: true)
        --mdns <mdns>                                Allow MDNS (default: false)
        --metrics-address <metrics-address>
            Address used for metrics collection server. By defaults binds on localhost on port 6116

        --req-window <req-window>                    Number of tipsets requested over chain exchange (default is 200)
    -r, --rpc <rpc>                                  Allow RPC to be active or not (default: true)
        --rpc-address <rpc-address>                  Address used for RPC. By defaults binds on localhost on port 1234
        --target-peer-count <target-peer-count>      Amount of Peers we want to be connected to (default is 75)
        --tipset-sample-size <tipset-sample-size>
            Number of tipsets to include in the sample that determines what the network head is

    -t, --token <token>                              Client JWT token to use for JSON-RPC authentication

ARGS:
    <cmd>
```